server/dummy-data.json

```diff
@@ -0,0 +1,28 @@
+[
+  {
+    "name": "Jamie's Bakery",
+    "address": "123 Fake Street",
+    "howToOrder": "Online",
+    "website": "https://jamies-bakery.com/",
+    "openingHours": "Online between 8am - 8pm online for next day delivery at the latest",
+    "notes": "Still delivering fresh made bread and delicious treats every day",
```

Comment:
I would change "notes" into the following 2:
```
"generalDescription": "Jamie's is your neighbourhood bakery specialising in sourghdough and cakes"
"covidService": "We are still delivering fresh made bread and delicious treats every day"
```

Reply:
Ok cool, will we have separate columns for these in the spreadsheet? Or will we need to split it up somehow?

Reply:
I will make separate columns in the spreadsheet to divide up the ones we already have. I actually have it that way in the template I send to new businesses who want to be listed.  I think this means when this is over - we can take out the Covid bit quite easily and it becomes a useful local resource!  Also, it means that we can really highlight the new services they are providing due to the lockdown.

Reply:
Small thing, there is a slight discrepancies between the json here and the google sheets:
- `generalDescription` is Short Description
- `covidService` is Lockdown Services
Again, small thing and am happy with either key but we should update so both are the same.

Reply:
Thanks for that - I think I realised when I was putting it in the sheet we need it be understandable for people entering info.

Can we please change to `shortDescription` and `lockdownServices`

Reply:
Also, businesses might have multiple social media accounts with variations on their name so that `socialMedia `will need to be divided up into:

```
instagramHandle
facebookHandle
twitterHandle
```

Reply:
Final addition, I've given Delivery and Collection each a tick box column in the spreadsheet as this is important info and should be separate to tags (which is how it's set up now).  Happy to hear of any other way to do this - take a look at sheet and see what you think!

So we also need (I think):
```
"delivery":"true"
"collection":"false"
```

Reply:
All sounds good! Maybe we can make `socialMedia` an object?

Will try and make updates but a bit swamped at the moment. If I'm taking too long, feel free to pull down branch and update PR

Reply:
I will try to update now 😊